assets/css/nav/_top_nav.scss

```diff
@@ -56,4 +56,13 @@
 
 .c-top-nav__popover {
   z-index: map-get($z-page-layout-context, "user-info-popover");
+
+  // Needed because we don't yet pull in Reboot which resets browser hr style
+  // Can't just set border: 0 like Reboot does beause this rule has high
+  // specificity and would override the dropdown divider border-top style
+  hr {
+    border-left: 0;
+    border-right: 0;
+    border-bottom: 0;
+  }
```

Comment:
I missed that note from design - was that just for the internal divider or also the border around the dropdown?

Reply:
I don't think it was written down, but as I understood it, we're likely going to set bootstraps `$border-color` to `tokens_2024.$gray-400`, which would apply it to the `<hr/>` as well. I don't think it's worth guessing what we'll do in the future though, it's probably better to stick to defaults and wait for the reboot/typo/colors work to be done.

Reply:
> I don't think it's worth guessing what we'll do in the future though, it's probably better to stick to defaults and wait for the reboot/typo/colors work to be done.

Yeah unless design specifically requests it I think this is the safer option for now.

Reply:
But I still need the resetting of borders to `0` on `hr`, regardless of the color. could that be moved into the global namespace, or have `c-nav-menu` added to the list of allowed parents on this selector?

I could otherwise open a PR to add this for just the `c-nav-menu` but then we need to remove the `hr` styles in two places once we have the reboot.

Reply:
Yes, sorry, I'm working on some other stuff and haven't gotten to addressing that part but it's on my mind.

Reply:
Okay, thanks!!

Reply:
As for the border reset stuff... I'm kind of struggling to identify anywhere that would be good to put it without potentially impacting other `<hr/>` elements we already have, though I'm open to suggestions.

Reply:
What if we created a `bootstrap_polyfills` or something and then added this rule?

```
.c-top-nav__popover, .c-nav-menu {
  hr {
    border-left: 0;
    border-right: 0;
    border-bottom: 0;
  }
}
```

Reply:
Oh, I like that, and it makes it very clear what it's there for so that we know to remove it as we add more Bootstrap stuff! How about I add it as `css/bootstrap/_polyfills.scss` to work it into our existing folder structure?

Reply:
Well I went ahead and did it, let me know what you think: 6a20a8bd

Reply:
Love it!